src/TrackGenerator.cpp

```diff
@@ -747,13 +755,13 @@ void TrackGenerator::initializeTracks() {
 
       /* If Track points to the upper right */
       if (i < _num_azim / 2)
-        _tracks[i][_num_x[i]+j].getStart()->setCoords(0,
-                                     dy_eff[i] * (0.5 + j));
+        _tracks[i][_num_x[i]+j].getStart()->setCoords
+          (0, dy_eff[i] * (0.5 + j), z);
```

Comment:
That is correct. I'll also make the indent 4 lines. Interestingly, the google C++ style guide recommends a 4 line indent if no function arguments are given on the line that starts the function, but recommends aligning the arguments with the start of the parentheses if any arguments are given on the line that starts the function.